### PR TITLE
Support for out of box dashboards and include hostmetrics dashboard

### PR DESCRIPTION
**Pull request by prashant-shahi**
Related to https://github.com/SigNoz/signoz/issues/1801

Only difference in the included Hostmetrics dashboard JSON is that it includes `uuid` key.
Which makes sure that same dashboard is not loaded for every query-service restart.

Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

**Review by makeavish — DISMISSED** [review]

File: pkg/query-service/app/dashboards/provision.go [hunk 1]
```diff
@@ -37,13 +37,17 @@ func readCurrentDir(dir string) error {
 			continue
 		}
 
-		_, apiErr := GetDashboard(data["uuid"].(string))
-		if apiErr == nil {
-			zap.S().Infof("Creating Dashboards: Error in file: %s\t%s", filename, "Dashboard already present in database")
-			continue
+		zap.S().Infof("Dashboard: file: %s\t uuid: %s", filename, data["uuid"].(string))
+
+		if dashboardUuid, ok := data["uuid"]; ok {
```
Comment (srikanthccv) [inline pkg/query-service/app/dashboards/provision.go:42]: If `data["uuid"]` doesn't exist in the map, I believe `data["uuid"].(string)` will panic, please check once.

Comment (prashant-shahi) [reply pkg/query-service/app/dashboards/provision.go]: @srikanthccv  updated

**Review by palashgdev — CHANGES_REQUESTED** [review]

**Comment by ankitnayan** [comment]
we should move the promQL queries to use the new query builder APIs

**Comment by prashant-shahi** [comment]
> we should move the promQL queries to use the new query builder APIs

@ankitnayan yes, will work on migrating dashboards. Last time, I had tried there were few queries which caused some issues.

@srikanthccv was it something with spatial dimension? I guess it was with `SUM` operator.

**Comment by srikanthccv** [comment]
It was something along the line of two aggregate operators that we don't have on the dropdown list. Not sure what exactly it was but we can write ClickHouse queries if it is not possible with builder now while we figure out how to address it with query builder.

**Comment by makeavish** [comment]
@prashant-shahi Please move promQL queries to query builder. The ones which can't be moved, create an issue for that.

**Comment by prashant-shahi** [comment]
> @prashant-shahi Please move promQL queries to query builder. The ones which can't be moved, create an issue for that.

can you create issue about this in [Dashboards repository](https://github.com/SigNoz/dashboards)?

**Comment by prashant-shahi** [comment]
Closing this PR as it is stale now.

**Comment by srikanthccv** [comment]
> Also, the queries involving multiple aggregated queries like CPU usage calculation had issues.


Here is the equivalent query in builder.

Screenshot 2023-10-17 at 2 18 02 PM

**Comment by makeavish** [comment]
> > Also, the queries involving multiple aggregated queries like CPU usage calculation had issues.
> 
> Here is the equivalent query in builder.
> 
> Screenshot 2023-10-17 at 2 18 02 PM

@prashant-shahi

**Pull request closed without merging**